code/modules/species/station/xenomorph_hybrids/hybrid_abilities.dm

```diff
@@ -67,9 +67,65 @@
 				E.status &= ~ORGAN_BROKEN
 				to_chat(O, SPAN_NOTICEALIEN("You mend the bone in your [E]"))
 				return//fix one then stop, trigger again to mend more
+#define XENOHYBRID_SNEAK_ABILITY "/datum/ability/species/xenomorph_hybrid/sneak"
```

Comment:
trait sources should be in `__DEFINES`, not here, 
and if you don't want to define just use the typepath, not a string

technically, it's better to use strings, but definitely don't use a typepath as a string unless there's a good reason.